src/Features/Core/Portable/Wrapping/InitializerExpression/InitializerExpressionCodeActionComputer.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Wrapping.InitializerExpression
+{
+    internal abstract partial class AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax>
+    {
+        /// <summary>
+        /// Class responsible for actually computing the entire set of code actions to offer the user.
+        /// </summary>
+        private sealed class InitializerExpressionCodeActionComputer : AbstractSeparatedListCodeComputer<AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax>>
+        {
+            private readonly SyntaxTrivia _braceIndentationTrivia;
+            private readonly bool _doMoveOpenBraceToNewLine;
+
+            public InitializerExpressionCodeActionComputer(
+                AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax> service,
+                Document document, SourceText sourceText, DocumentOptionSet options,
+                TListSyntax listSyntax, SeparatedSyntaxList<TListItemSyntax> listItems,
+                bool doMoveOpenBraceToNewLine, CancellationToken cancellationToken)
+                : base(service, document, sourceText, options, listSyntax, listItems, cancellationToken)
+            {
+                var generator = SyntaxGenerator.GetGenerator(OriginalDocument);
+
+                _braceIndentationTrivia = generator.Whitespace(GetBraceTokenIndentation());
+                _doMoveOpenBraceToNewLine = doMoveOpenBraceToNewLine;
+            }
+
+            private string GetBraceTokenIndentation()
+            {
+                var previousToken = _listSyntax.GetFirstToken().GetPreviousToken();
+                return GetIndentationAfter(previousToken, Formatting.FormattingOptions.IndentStyle.Block);
```

Comment:
comment in code while "IndentStyle.Block" is appropriate here.

Reply:
> 
> 
> Note: there are nullable warnings in your code, so teh build is failing. I can help with this if you want!



> 
> 
> Note: there are nullable warnings in your code, so teh build is failing. I can help with this if you want!

I took a shot at fixing it. If what I did doesn't make sense I am definitely open to you helping or talking through how it should be fixed and doing it myself.